### PR TITLE
Add stock locations

### PR DESCRIPTION
**Pull request by danisal**
- Update generate-graphql-types script to use env file

TODO: we need to wait for the prod API to be update with stock locations

### CODE REVIEW

File: src/bootstrap-tenant/__dev__bootstrap.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { bootstrapTenant } from './index'
 import { EVENT_NAMES, Status } from './bootstrapper'
 
 function bootstrap() {
-  const tenantIdentifier = 'hkn-examples'
+  const tenantIdentifier = 'salvado-inc.'
```
Comment (nerdenough) [inline src/bootstrap-tenant/__dev__bootstrap.ts:11]: Is this supposed to be changed?

File: src/bootstrap-tenant/__dev__create-spec.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { resolve } from 'path'
 import { createJSONSpec } from './index'
 
 async function createSpec() {
-  const tenantIdentifier = 'hkn-examples'
+  const tenantIdentifier = 'salvado-inc.'
```
Comment (nerdenough) [inline src/bootstrap-tenant/__dev__create-spec.ts:10]: Same here

**Comment by danisal** [comment]
@hakonkrogh do you think this is good to go? so we can release stock locations tomorrow

**Comment by hakonkrogh** [comment]
> @hakonkrogh do you think this is good to go? so we can release stock locations tomorrow

I'd say so yes 👍

**Pull request merged**